src/metabase/notification/payload/impl/dashboard.clj

```diff
@@ -42,7 +42,8 @@
                                 :color_text_light  channel.render/color-text-light
                                 :color_text_medium channel.render/color-text-medium}
        :parameters             parameters
-       :dashboard_subscription dashboard_subscription})))
+       :dashboard_subscription dashboard_subscription
+       :disable_links          (:disable_links dashboard_subscription)})))
```

Comment:
This the payload that will be accessible in the template `src/metabase/channel/email/dashboard_subscription.hbs`